docs/index.rst

```diff
@@ -1,711 +1,14 @@
-       "@id": "#sdqa_Rating_CcdVisit.metricValue"
-       datatype: double
-       description: Value of this SDQA metric.
-       mysql:datatype: DOUBLE
-     - name: metricSigma
-       "@id": "#sdqa_Rating_CcdVisit.metricSigma"
-       datatype: double
-       description: Uncertainty of the value of this metric.
-       mysql:datatype: DOUBLE
-     primaryKey: "#sdqa_Rating_CcdVisit.sdqa_ratingId"
-     constraints:
-     - name: UQ_sdqaRatingCcdVisit_metricId_ccdVisitId
-       "@id": "#UQ_sdqaRatingCcdVisit_metricId_ccdVisitId"
-       "@type": Unique
-       columns:
-       - "#sdqa_Rating_CcdVisit.sdqa_metricId"
-       - "#sdqa_Rating_CcdVisit.ccdVisitId"
-     indexes:
-     - name: IDX_sdqaRatingCcdVisit_metricId
-       "@id": "#IDX_sdqaRatingCcdVisit_metricId"
-       columns:
-       - "#sdqa_Rating_CcdVisit.sdqa_metricId"
-     - name: IDX_sdqaRatingCcdVisit_thresholdId
-       "@id": "#IDX_sdqaRatingCcdVisit_thresholdId"
-       columns:
-       - "#sdqa_Rating_CcdVisit.sdqa_thresholdId"
-     - name: IDX_sdqaRatingCcdVisit_ccdVisitId
-       "@id": "#IDX_sdqaRatingCcdVisit_ccdVisitId"
-       columns:
-       - "#sdqa_Rating_CcdVisit.ccdVisitId"
-     mysql:engine: MyISAM
-
-   - name: sdqa_Threshold
-     "@id": "#sdqa_Threshold"
-     description: Version-controlled metric thresholds. Total number of these records
-       is approximately equal to 30 x the number of times the thresholds will be changed
-       over the entire period of LSST operations (of order of 100), with most of the
-       changes occuring in the first year of operations.
-     columns:
-     - name: sdqa_thresholdId
-       "@id": "#sdqa_Threshold.sdqa_thresholdId"
-       datatype: short
-       description: Primary key.
-       mysql:datatype: SMALLINT
-     - name: sdqa_metricId
-       "@id": "#sdqa_Threshold.sdqa_metricId"
-       datatype: short
-       description: Pointer to sdqa_Metric table.
-       mysql:datatype: SMALLINT
-     - name: upperThreshold
-       "@id": "#sdqa_Threshold.upperThreshold"
-       datatype: double
-       description: Threshold for which a metric value is tested to be greater than.
-       mysql:datatype: DOUBLE
-     - name: lowerThreshold
-       "@id": "#sdqa_Threshold.lowerThreshold"
-       datatype: double
-       description: Threshold for which a metric value is tested to be less than.
-       mysql:datatype: DOUBLE
-     - name: createdDate
-       "@id": "#sdqa_Threshold.createdDate"
-       datatype: timestamp
-       description: Database timestamp when the record is inserted.
-       value: CURRENT_TIMESTAMP
-       mysql:datatype: TIMESTAMP
-     primaryKey: "#sdqa_Threshold.sdqa_thresholdId"
-     indexes:
-     - name: IDX_sdqaThreshold_metricId
-       "@id": "#IDX_sdqaThreshold_metricId"
-       columns:
-       - "#sdqa_Threshold.sdqa_metricId"
-     mysql:engine: MyISAM
+Felis provides a system for describing astronomical data catalogs with a user-friendly YAML data format.
```

Comment:
Add:
"It covers not only the metadata required to create database schemas, but also a wide range of additional metadata associated with IVOA standards and other applications."